pages/pil/unscoped/courses/routers/delete-training-course.js

```diff
@@ -0,0 +1,22 @@
+const { Router } = require('express');
+const { merge } = require('lodash');
+const content = require('../read/content');
+
+module.exports = () => {
+  const app = Router({ mergeParams: true });
+
+  app.post('/', (req, res, next) => {
+    const params = {
+      method: 'DELETE'
+    };
+    req.api(`/establishment/${req.establishmentId}/training-course/${req.trainingCourseId}`, params)
+      .then(() => {
+        res.locals.static.content = merge({}, res.locals.static.content, content);
```

Comment:
Setting `locals` on a POST is generally redundant (unless something weird is occurring) since it doesn't render.

Reply:
it's for the notification - the content is set from locals

Reply:
Ahhh, I had it in my head that the content for notifications was loaded at render-time. I checked and I'm wrong.